Sources/Internal/CodableTimestamp.swift

```diff
@@ -52,10 +52,9 @@ extension CodableTimestamp {
       .firstMatch(in: timestampString, range: NSRange(location: 0,
                                                       length: timestampString.count)) !=
       nil else {
-      FirebaseLogger.dataConnect
-        .error(
-          "Timestamp string format doesn't support."
-        )
+      DataConnectLogger.error(
+        "Timestamp string \(timestampString) format doesn't support."
```

Comment:
nit: If it's the format that is not supported, I think could be re-worded to be a little clearer.
```suggestion
        "Timestamp string format \(timestampString) is not supported."
```